### PR TITLE
Added methods for Messages

### PR DESCRIPTION
**Pull request by Latand**
Added methods for Messages:

addChatUser
allowMessagesFromGroup
createChat
delete
deleteChatPhoto
deleteConversation
denyMessagesFromGroup
edit
editChat
getByConversationMessageId
getById
getChat
getChatPreview
send

### CODE REVIEW

File: avkapi/methods/base.py [hunk 1]
```diff
@@ -21,10 +31,11 @@ def __init__(self, session, access_token, api_version):
         parameters['v'] = self._api_version
         p = {k: v for k, v in parameters.items() if v is not None}
         link = f'https://api.vk.com/method/{method_name}'
-
+        print(p)
```
Comment (Olegt0rr) [inline avkapi/methods/base.py:34]: Please remove prints. You can use debug level of logging here

File: avkapi/types/keyboard.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+
+@dataclass
+class ListOfButtons:
+    buttons: typing.List
+    payloads: typing.List
+    row_sizes: typing.List[int]
+    colours: typing.List = None
+    """
+    Использование:
```
Comment (Olegt0rr) [inline avkapi/types/keyboard.py:116]: Please use English language only

File: examples/send_message.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+VK_CALLBACK_SECRET_KEY = None
+VK_ACCESS_TOKEN = "6dd495521793844f17510f6359445288fc6cc3dae00de6e1c6e8c173e7854322b3f989d58b20b8e1a3af8"
+
+loop = asyncio.get_event_loop()
+vk = VK(confirmation_code=VK_CONFIRMATION_CODE,
+        secret_key=VK_CALLBACK_SECRET_KEY,
+        access_token=VK_ACCESS_TOKEN,
+        loop=loop)
+
+keyboard = ListOfButtons(buttons=["Кнопка Белая", "Кнопка Синяя", "Кнопка Красная", "Кнопка Зеленая"],
```
Comment (Olegt0rr) [inline examples/send_message.py:15]: Non pythonic style has complicated logics. Please, use Button(**params) to create button.

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,14 @@ draft*
 config*
 venv/
 .idea/
+*.pyc
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (JrooTJunior) [inline .gitignore]: Remove git merge parts

File: avkapi/dispatcher/dispatcher.py [hunk 5]
```diff
@@ -10,13 +10,17 @@
 from ..types import Message, MessageType
 from ..utils import context
 from ..vk import VK
+import typing
+from ..utils.mixins import ContextInstanceMixin, DataMixin
+import itertools
+import asyncio
```
Comment (JrooTJunior) [inline avkapi/dispatcher/dispatcher.py:16]: Reorder imports [by following PEP-8 guidelines](https://www.python.org/dev/peps/pep-0008/#imports)

File: avkapi/dispatcher/dispatcher.py [hunk 6]
```diff
@@ -97,6 +102,39 @@ def decorator(callback):
 
         return decorator
 
+    def register_callback_query_handler(self, callback, *custom_filters, state=None, run_task=None, **kwargs):
+        """
+        Register handler for callback query
+        Example:
+        .. code-block:: python3
+            dp.register_callback_query_handler(some_callback_handler, lambda callback_query: True)
+        :param callback:
```
Comment (JrooTJunior) [inline avkapi/dispatcher/dispatcher.py:111]: Add whitespace (1 line between description and argument descriptions)

File: avkapi/dispatcher/dispatcher.py [hunk 7]
```diff
@@ -97,6 +102,39 @@ def decorator(callback):
...
+        self.callback_query_handlers.register(self._wrap_async_task(callback, run_task), filters_set)
+
+    def callback_query_handler(self, *custom_filters, state=None, run_task=None, **kwargs):
+        """
+        Decorator for callback query handler
+        Example:
+        .. code-block:: python3
+            @dp.callback_query_handler(lambda callback_query: True)
+            async def some_callback_handler(callback_query: types.CallbackQuery)
+        :param state:
```
Comment (JrooTJunior) [inline avkapi/dispatcher/dispatcher.py:126]: Whitespace

File: avkapi/types/user.py [hunk 8]
```diff
@@ -0,0 +1,308 @@
...
+    common_count: base.Integer = fields.Field()
+    connections: Connections = fields.Field(base=Connections)
+    contacts: Contacts = fields.Field(base=Contacts)
+    counters: Counters = fields.Field(base=Counters)
+    country: Country = fields.Field(base=Country)
+    crop_photo: CropPhotoInfo = fields.Field(base=CropPhotoInfo)
+    domain: base.String = fields.Field()
+    education: Education = fields.Field(base=Education)
+    exports: base.String = fields.Field()  # Not sure which type. Not specified
+    # first_name_{case}: base.String = fields.Field()  ??????
```
Comment (JrooTJunior) [inline avkapi/types/user.py:264]: what values can `{case}` accept?

**Review by JrooTJunior — CHANGES_REQUESTED** [review]

**Pull request closed without merging**